.travis.yml

```diff
@@ -1,13 +1,11 @@
 language: php
 
 php:
-  - 5.6
```

Comment:
This to me indicates we are dropping php 5.6 support.

If that is the case, then how about removing it from the `composer.json` as well?

If we are not yet dropping 5.6 support, then how about we keep testing it, untill we are 100% sure that support is dropped.

Reply:
This version of Profile will only work on PHP 7.0, so removing the support in the composer.json is a good suggestion.